CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19.11.2
+
+* CLI
+  * Updated `uninject` command to uninject namespace resources
```

Comment:
TIOLI: maybe avoid the second "uninject" and just do: `to work with namespace resources`. We already can tell  that the purpose of the command is to uninject things :)